packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -46,6 +46,13 @@ export interface ITable extends IResource {
  * @see https://docs.aws.amazon.com/athena/latest/ug/encryption.html
  */
 export enum TableEncryption {
+  /**
+   * Previous option. Tables can not be unencrypted now.
```

Comment:
Given that `aws-glue` is still alpha, we should consider removing this property altogether, so we don't carry this dead weight up to graduation.

I'd like to hear what other contributors think.

Reply:
It sounds fair to me.
Waiting for some more comments and I'll proceed.

Reply:
Agreed on this.

Reply:
Removed `UNENCRYPTED` property.